AggregateMetrics/AggregateMetrics/AzureWebApp/PerformanceCollectorModule.cs

```diff
@@ -21,14 +24,40 @@ public class PerformanceCollectorModule : ITelemetryModule
                                                                 @"\Processor(_Total)\% Processor Time"
                                                             };
 
-        public IList<PerformanceCounterCollectionRequest> Counters { get; private set; }
+        public List<PerformanceCounterCollectionRequest> Counters { get; private set; }
```

Comment:
This kind of return type needs to be interface and not concrete, furthermore, List is specifically supposed to be for internal use and not exposed directly (there is an FxCop rule for this).